docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -104,25 +156,18 @@ in *any* synthetics tools.
 Instead, set up limited demo accounts, or fake credit cards with limited functionality.
 ====
 
-Use environment variables to handle any values needing encryption at rest. 
-The example below uses the syntax `${URL}` to reference a variable named `URL` defined in the environment:
+If you are managing monitors with projects, you can use environment variables
+in your `synthetics.config.ts` or `synthetics.config.js` file.
 
-[source,yaml]
+The example below uses `process.env.MY_URL` to reference a variable named `MY_URL`
+defined in the environment and assigns its value to a param. That param can then
+be used in both lightweight and browser monitors that are managed in the project:
+
+[source,js]
 ----
-- name: Todos
-  id: todos
-  type: browser
-  schedule: "@every 3m"
-  tags: todos-app
-  params:
-    url: ${URL}
-  source:
-    inline:
-      script: |-
-      step("load homepage", async () => {
-        await page.goto(params.url);
-      });
-      step("hover over products menu", async () => {
-        await page.hover('css=[data-nav-item=products]');
-      });
+export default {
+  params: {
+    my_url: process.env.MY_URL
+  }
+};
```

Comment:
@paulb-elastic @dominiqueclarke let me know what you think of this approach.

Reply:
Looks like: 

<img width="876" alt="Screenshot 2023-05-18 at 11 53 51 AM" src="https://github.com/elastic/observability-docs/assets/10479155/84a984ce-7ecd-4217-9377-e40f9d483973">

Reply:
LGTM